platform_versioned_docs/version-24.1/data_studios/index.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: "Overview"
+description: "Data Studios public preview."
+date: "1 Oct 2024"
+tags: [data, studios]
+---
+
+Data Studios is a unified platform where you can host a combination of container images and compute environments for interactive tertiary analysis using your preferred tools, like JupyterLab and RStudio Notebooks, Visual Studio Code IDEs, or Xpra remote desktops. Each data studio session is an individual interactive environment that encapsulates the live environment for dynamic data analysis.
```

Comment:
```suggestion
Data Studios is a unified platform where you can host a combination of container images and compute environments for interactive analysis using your preferred tools, like JupyterLab and RStudio Notebooks, Visual Studio Code IDEs, or Xpra remote desktops. Each data studio session is an individual interactive environment that encapsulates the live environment for dynamic data analysis.
```

I've found in the past that the meaning of 'tertiary' analysis isn't obvious to everyone.